### PR TITLE
Table view fix

### PR DESCRIPTION
**Pull request by TG908**
-fix for white borders around cells
-removed separators

### CODE REVIEW

**Comment by TG908** [comment]
Something is wrong with my build settings...

File: TUM Campus App/CardViewController.swift [hunk 1]
```diff
@@ -75,7 +74,7 @@ extension CardViewController {
         tableView.rowHeight = UITableViewAutomaticDimension
         imageView.clipsToBounds = true
         tableView.tableFooterView = UIView(frame: CGRect.zero)
-        tableView.separatorColor = UIColor.clear
+        tableView.separatorStyle = .none
```
Comment (mathiasquintero) [inline TUM Campus App/CardViewController.swift]: This looks familiar. Is this the same change as in the other PR for the spinner fix?

**Comment by mathiasquintero** [comment]
This PR doesn't do what it says on the title

**Comment by TG908** [comment]
For me it does the following: -fix for white borders around cells
-removed separators

**Comment by mathiasquintero** [comment]
Look at the diff

**Comment by TG908** [comment]
I had trouble with the build settings so i had to merge the new build settings from my master to this branch and i also reverted it once. Maybe thats why i doesnt show up in the diff

**Comment by TG908** [comment]
Weird... I will create a new PR wiht a proper diff...

**Comment by TG908** [comment]
The fix for the table view was somehow also in another PR and when it got merged the diff disappeared...

**Comment by mathiasquintero** [comment]
Not really a mystery. The other branch probably used this one

**Pull request closed without merging**